README.md

```diff
@@ -1,33 +1,82 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Behaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidly prioritising them based on impact they may have.
```

Comment:
Suggest using "problems" instead of "issues" because I think it makes more sense to "solve a problem" than to "solve an issue". Also removed "may" from the second part.

```suggestion
- We aim to solve problems rapidly, prioritising them based on their impact.
```